### ISSUE TITLE
[Top Bar] TopBar misses ACTIVE feature

### ISSUE THREAD

**Issue by karleKremen** [zurb/foundation-sites#8040]
**What did you expect to happen?**

I use FOundation on my site and I need to have ACTIVE class for active menu entry.

**What happened instead?**

I found that TopBar hasn't documentated way to do it!

**Test case**

[link](http://kremenlib.cf/about/#!)

**Comment by zurbchris** [comment]
Top Bar is now just a container with some styling and no scripts associated with it. You have a dropdown menu included, but no nested menus with which to trigger the `js-dropdown-active` state class. 

Perhaps you could clarify a little better what behavior you're expecting, and we can help you get there.

**Comment by rafibomb** [comment]
The Foundation 5 top-bar has an active class. It is not a JS state but rather a simple class.

```
.top-bar-section ul li.active > a {
      background: #008CBA;
      color: #FFFFFF; }
      .top-bar-section ul li.active > a:hover {
        background: #0078a0;
        color: #FFFFFF; }
```

Do see a need to add the class if the CSS is simple to write and you will likely override the defaults?

**Comment by zurbchris** [comment]
That class is still added via JS, which is not present in F6 because Top Bar has no associated scripts. This can be done, rather easily, with Toggler.

**Comment by rafibomb** [comment]
This is the CSS necessary:

```
.menu > li.active {
  background: red;
  color: #fff;
}
```

http://codepen.io/rafibomb/pen/vLrGwg

In a static site, the active class is added manually, right?

**Comment by zurbchris** [comment]
As in hard-coded like you did in your pen? Yes.

**Comment by karleKremen** [comment]
@rafibomb, thanx, I added nearle same code by myself. But... Foundation 5 had so many really great features, why Foundation 6 doesn't? Now I think to migrate back to F5!

**Comment by gakimball** [comment]
More likely we'll add an active class to menu, not top bar. We actually had to write our own for the table of contents in the docs, so it seems reasonable.

**Comment by gakimball** [comment]
Implemented in the `6.2` branch: 9c1b1042da931f25a80fe61ff32f99828b5f7b66

**Comment by SalahAdDin** [comment]
:+1:
